tests/twithoptions.nim

```diff
@@ -0,0 +1,10 @@
+import  optionsutils, options
+
+proc a(a,b: Option[int], c = 100){.withSome.}= echo a, " ", b, " ", c
+
+proc b(a: Option[int]){.withNone.}= echo "Only Nones Work!"
+
+a(some(10), none(int), 300) #Doesn't Print
```

Comment:
Rather than manual comments you should return the strings and then use `assert` or `check` (from the `unittest` module) to verify the outputs.